microcosm_pubsub/main.py

```diff
@@ -18,6 +21,7 @@ class SimpleSchema(PubSubMessageSchema):
     MEDIA_TYPE = "application/vnd.globality.pubsub.simple"
 
     message = fields.String(required=True)
+    timestamp = fields.Float(required=True)
```

Comment:
I modified the example schema to have a timestamp so we can easily trace delivery time.